packages/bpk-component-page-indicator/src/BpkPageIndicator.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { cssModules } from '../../bpk-react-utils';
+
+import NavButton, { DIRECTIONS } from './NavButton';
+import STYLES from './BpkPageIndicator.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+const DISPLAYED_TOTAL = 5;
+const START_SCROLL_INDEX = Math.floor(DISPLAYED_TOTAL / 2);
+
+export type Props = {
+  className: ?string,
+  showNav: ?boolean,
+  currentIndex: number,
+  totalIndicators: number,
+  onClick: ?() => void,
+  indicatorLabel: string,
+  prevNavLabel: string,
+  nextNavLabel: string,
+  overImage: ?boolean,
+};
```

Comment:
I spoke to the team to align on the API so that we can replicate the same for apps and a good idea came up

Could we change this from being a boolean and switch to using a prop called `style`, which would then be an enum of two styles `'default'` and `'overImage'`?

This makes it expandable for the future and moves away from booleans to set the style

Reply:
Hi I've exported the props `style` with two styles - `default` and `overImage`. Thank you for your comment!

Reply:
`style` can be confused with native `style` property, maybe align it with the `Badge` or `Chip` API naming, using `type`? Or start here aligning all components to `variant` prop name for representation variations (because f.e. `button` should have native `type` prop passed through) 👀

Reply:
Let me take this back to the team to align the API again.

This could prove useful using this to block the native prop as the component has been set and designed so you should not be looking to change the styles either 🙂